components/d2l-inner-module.js

```diff
@@ -245,5 +250,10 @@ class D2LInnerModule extends ASVFocusWithinMixin(PolymerASVLaunchMixin(Completio
 		return hideDescription ? 'hide-description' : '';
 
 	}
+
+	_getHasActiveChild(entity, currentActivity) {
+		const result = entity && entity.entities.filter(subEntity => subEntity.href === currentActivity) || [];
```

Comment:
Could we update this to some?
```suggestion
		return entity && entity.entities.some(subEntity => subEntity.href === currentActivity) || false;
```

Reply:
Done.